lib/ricohapi/mstorage/client.rb

```diff
@@ -9,6 +9,8 @@ class Client
       class Error < StandardError; end
       class Unauthorized < Error; end
 
+      User_meta_regex = /^user\.([A-Za-z0-9_\-]{1,256})$/
```

Comment:
If you define `User_meta_regex` as a constant, I think you should define it `USER_META_REGEX`.